src/stravalib/tests/integration/test_client.py

```diff
@@ -488,6 +488,8 @@ def _call_and_assert():
     (
         ({}, {}, None),
         ({"activity_type": "run"}, {"type": "run"}, None),
+        ({"sport_type": "TrailRun"}, {"sport_type": "TrailRun"}, None),
+        ({"sport_type": "funrun"}, {"sport_type": "funrun"}, ValueError),
```

Comment:
This is good, but just as with the test for `create_activity()`, we should add the case where both `activity_type` and `sport_type` are provided (and expect the same result as in the other test).